csharp/src/ProtocolBuffers.Test/CodedInputStreamTest.cs

```diff
@@ -232,66 +230,26 @@ public void DecodeZigZag64()
             Assert.AreEqual(0x7FFFFFFFFFFFFFFFL, CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFEL));
             Assert.AreEqual(unchecked((long) 0x8000000000000000L), CodedInputStream.DecodeZigZag64(0xFFFFFFFFFFFFFFFFL));
         }
-        /*
+        
         [Test]
-        public void ReadWholeMessage()
+        public void ReadWholeMessage_VaryingBlockSizes()
         {
-            TestAllTypes message = TestUtil.GetAllSet();
+            TestAllTypes message = GeneratedMessageTest.GetSampleMessage();

```

Comment:
Most of the code within it was becoming useless - it ended up really only being useful for getting a sample message. I can move that back in if you think it's better there though. (Probably not today or tomorrow - I'm in Zurich at the moment.)